portal-impl/src/com/liferay/portal/security/membershippolicy/RoleMembershipPolicyFactoryUtil.java

```diff
@@ -61,19 +54,7 @@ private static class RoleMembershipPolicyTrackerCustomizer
 		public RoleMembershipPolicy addingService(
 			ServiceReference<RoleMembershipPolicy> serviceReference) {
 
-			RoleMembershipPolicy roleMembershipPolicy =
-				_bundleContext.getService(serviceReference);
-
-			if (PropsValues.MEMBERSHIP_POLICY_AUTO_VERIFY) {
```

Comment:
@JanisZhang What do you mean by "this can be implemented in actual impl"?

Reply:
@dantewang 

Hi Dante, this property will be checked when portal starts or deploy webhook.

Since We currently only have dummy service, so previously I think it can be eliminated here and implement in the actual impl class, like `DefaultSiteMembershipPolicy`.

However, I just noticed after I removed `DefaultSiteMembershipPolicy` from spring.xml,method `afterPropertiesSet` will not execute.

And here is another problem, We currently have 4 types of membershipPolocy: Role,Site,Organization and User. Currently only Role and User types(utilizing `RoleMembershipPolicyFactoryUtil` and `UserGroupMembershipPolicyFactoryUtil` ) have this verify invocation when adding service. I've opened a PTR to the Echo team. After that, I will find a way to deal with this.

Thanks.

Reply:
@dantewang 

Hi Dante, this property will be checked when portal starts or deploy webhook.

Since We currently only have dummy service, so previously I think it can be eliminated here and implement in the actual impl class, like `DefaultSiteMembershipPolicy`.

However, I just noticed after I removed `DefaultSiteMembershipPolicy` from spring.xml,method `afterPropertiesSet` will not execute.

And here is another problem, We currently have 4 types of membershipPolocy: Role,Site,Organization and User. Currently only Role and User types(utilizing `RoleMembershipPolicyFactoryUtil` and `UserGroupMembershipPolicyFactoryUtil` ) have this verify invocation when adding service. I've opened a PTR to the Echo team. After that, I will find a way to deal with this.

Thanks.